es-design-system/package.json

```diff
@@ -6,15 +6,11 @@
         "dev": "env-cmd -f .env.local nuxt",
         "build": "nuxt generate",
         "build:dev": "npm run generate:dev && npm run deploy:dev",
-        "build:prod": "npm run generate:prod && npm run deploy:prod",
-        "generate:dev": "env-cmd -f .env.dev npm run build",
-        "generate:prod": "env-cmd -f .env.prod npm run build",
-        "bootstrap": "env-cmd -f .env.prod npm --prefix aws run cdk bootstrap",
-        "synth:dev": "env-cmd -f .env.dev npm run synth",
-        "synth:prod": "env-cmd -f .env.prod npm run synth",
+        "generate:dev": "npm run build",
```

Comment:
We don't have to address this in this PR, but the npm script names are a little confusing. I would expect "generate" to run `nuxt generate` and "build" to run `nuxt build`, ect.

Reply:
[ab0e276](https://github.com/EnergySage/es-ds/pull/666/commits/ab0e276ec07e3cf366eed33f56b61b58f2e44ec8)